lib/resources/r4/document_reference.yaml

```diff
@@ -186,8 +186,8 @@ fields:
      "attachment": {
             "contentType": "application/xml;charset=utf-8",
             "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
-            "title": "Summary Note",
-            "creation": "2020-01-15T14:35:29.000Z"
+            "title": "Height Weight Allergy Rule",
```

Comment:
same here???

Reply:
I will check with my team lead and update you. Thanks for input.